### PR TITLE
Doxygen Additions for mova2i.f orders.f pdsens.f pdseup.f putgb.f (part of #77)

### PR DESCRIPTION
**Pull request by BrianCurtis-NOAA**
Doxygen additions for:
- mova2i.f
- orders.f
- pdsens.f
- pdseup.f
- putgb.f

(part of #77 )

### CODE REVIEW

**Comment by BrianCurtis-NOAA** [comment]
Apologies for the one file with whitespace removals.

File: src/pdseup.f [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
+C> @param[out] KMEMBR (80) Bytes 77-86 (cluster membership section, if needed.).
+C> @param[in] ILAST Last byte to be unpacked (if greater/equal to first byte
+C> in any of four sections below, whole section is packed).
+C> @param[in] MSGA Full pds section, including new ensemble extension.
 C>
-C> USE PDSENS.F FOR PACKING PDS ENSEMBLE EXTENSION.
-C> SUBPROGRAM CAN BE CALLED FROM A MULTIPROCESSING ENVIRONMENT.
+C> @note Use pdsens() for packing pds ensemble extension.
+C> Subprogram can be called from a multiprocessing environment.
 C>
```
Comment (Hang-Lei-NOAA) [inline src/pdseup.f:25]: C> @author Zoltan Toth & Mark Iredell @date 1998-03-14

File: src/pdsens.f [hunk 2]
```diff
@@ -1,29 +1,27 @@
...
+C> @param[in] ILAST Last byte to be packed (if greater or equal to first byte
+C> in any of four sections above, whole section is packed).
+C> @param[out] MSGA - Full pds section, including new ensemble extension.
 C>
-C> USE PDSEUP.F FOR UNPACKING PDS ENSEMBLE EXTENSION.
-C> SUBPROGRAM CAN BE CALLED FROM A MULTIPROCESSING ENVIRONMENT.
+C> @note Use pdseup() for unpacking pds ensemble extension.
+c> subprogram can be called from a multiprocessing environment.
 C>
       SUBROUTINE PDSENS(KENS,KPROB,XPROB,KCLUST,KMEMBR,ILAST,MSGA)
```
Comment (Hang-Lei-NOAA) [inline src/pdsens.f:27]: C> @author Zoltan Toth & Mark Iredell @date 1998-03-14

**Comment by BrianCurtis-NOAA** [comment]
Thanks Hang. This is too funny. I keep telling myself when I am done to add the author to the last line, and I don't forget on every file. I need to start doing that at the beginning of my editing process.

**Comment by BrianCurtis-NOAA** [comment]
@Hang-Lei-NOAA mind reviewing again?

**Review by Hang-Lei-NOAA — APPROVED** [review]

**Pull request merged**